desktop/diskStorageManager.js

```diff
@@ -0,0 +1,49 @@
+// @flow
+import os from 'os';
+import disk from 'diskusage';
+import drivelist from 'drivelist';
+import { ipcConsts } from '../app/vars';
+
+const getMountPoint = (drive: any): string => {
+  if (drive.mountpoints && drive.mountpoints.length) {
+    return `${drive.mountpoints[0].path}${os.type() === 'win32' ? '\\' : ''}`;
+  }
+  return '/';
+};
+
+type Drive = {
+  id: string,
+  mountPoint: string,
+  label: string
+};
+
+class DiskStorageManager {
+  static getDriveList = async ({ event }: { event: any }) => {
+    drivelist.list((error, drives) => {
```

Comment:
you're stating that the function is async - then await usage is expected to async call synchronous but the code is written using callback and no sign of Promise whatsoever...